DQM/Integration/python/clients/beam_dqm_sourceclient-live_cfg.py

```diff
@@ -356,7 +356,7 @@
 
     DBParameters = cms.PSet(
                             messageLevel = cms.untracked.int32(0),
-                            authenticationPath = cms.untracked.string('')
+                            authenticationPath = cms.untracked.string('.')
```

Comment:
this change must be in all clients below, right?

Reply:
In particular beamfake misses the "dot"

Reply:
Hi @jfernan2, 
both "beamfake" clients alreay had the `authenticationPath = cms.untracked.string('.')` correctly set, that is why they don't appear between the changes of this PR.

Reply:
Hi @francescobrivio,
I can see that indeed `authenticationPath` variables are set to `'.'` but the same problem persists. After some debugging I can see that the problem is here: https://github.com/cms-sw/cmssw/blob/77bb7f0ba7cd3c1418c6a35f4b35cb32c2d712fb/CondCore/CondDB/src/ConnectionPool.cc#L106
`HOME` env variable is set to `/tmp` which is the reason for a crash. replacing the highlighted line with this: `authPath += ".";` fixes the issue.

Reply:
However, I would suggest to find out where the empty `authPath` is coming from and to eliminate that.

Reply:
@ggovi could you help me out in understanding where the empty authPath comes from?

Reply:
> However, I would suggest to find out where the empty `authPath` is coming from and to eliminate that.

I fully agree with you, about the fix you propose I would wait @ggovi to give me green light for it and then I can easily make it, but I am afraid it may mess up something else

Reply:
BTW, I think I have found where the empty authPath is coming from (GlobalTag), I have updated the PR with a possible fix.